pages.de/android/screencap.md/screencap.md

```diff
@@ -0,0 +1,9 @@
+# Bildschirmfotos
```

Comment:
```suggestion
# screencap
```
We don't translate the command name across our pages (with a few exceptions like this https://github.com/tldr-pages/tldr/pull/10840/files where the title isn't the command name as our linter doesn't accept symbols)